app/src/app/initApp.js

```diff
@@ -72,19 +73,25 @@ export default () => {
       }
     )
 
+    // workaround to solve invisible app problem when windows was closed on
+    // another monitor previously but Grape starts without the monitor now
+    if (state.dimensions.x < 0 || state.dimensions.x < 0) {
```

Comment:
I don't understand your comment and your code is broken.

Reply:
oh shit!